pkg/api/peer_req_resp.go

```diff
@@ -11,6 +11,7 @@ type Peer struct {
 	PeerAddresses   []string          `json:"peer-addresses"`
 	ClientAddresses []string          `json:"client-addresses"`
 	Online          bool              `json:"online"`
+	PID             string            `json:"pid,omitempty"`
```

Comment:
This should be an integer. Displaying it in CLI as not zero is a different problem.